commons-dlp/src/main/resources/db/changelog/exo-dlp.db.changelog-1.0.0.xml

```diff
@@ -30,11 +32,31 @@
         <constraints nullable="false"/>
       </column>
     </createTable>
-    <createSequence sequenceName="SEQ_DLP_QUEUE_ID" startValue="1"/>
   </changeSet>
 
   <changeSet author="exo-dlp" id="1.0.0-2" dbms="oracle,postgresql">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <sequenceExists sequenceName="SEQ_DLP_QUEUE_ID"/>
+      </not>
+    </preConditions>
     <createSequence sequenceName="SEQ_DLP_QUEUE_ID" startValue="1"/>
   </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+  <changeSet author="exo-dlp" id="1.0.0-3">
+    <createTable tableName="DLP_POSITIVE_ITEMS">
+      <column name="ITEM_ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="PK_DLP_ITEMS"/>
+      </column>
+      <column name="ITEM_REFERENCE" type="VARCHAR(50)"/>
+      <column name="ITEM_TYPE" type="VARCHAR(50)"/>
+      <column name="KEYWORDS" type="VARCHAR(50)"/>
+      <column name="DETECTION_DATE" type="TIMESTAMP"/>
+    </createTable>
+  </changeSet>
+
+  <changeSet author="exo-dlp" id="1.0.0-4" dbms="oracle,postgresql">
+    <createSequence sequenceName="SEQ_DLP_POSITIVE_ITEMS_ID" startValue="1"/>
+  </changeSet>
```

Comment:
This is not part of the fix. It should not be in 6.1.x

Reply:
@rdenarie PR updated, thanks for having checked it